{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--ace9a546-fe8d-47ad-9e91-21fe390b1de7",
    "created": "2024-08-13T16:09:54.933823Z",
    "modified": "2024-08-13T16:09:54.933827Z",
    "name": "IBM Security Guardium denial of service",
    "description": "IBM Security Guardium 10.6, 11.3, and 11.4 could allow an authenticated user to cause a denial of service due to due to improper input validation.  IBM X-Force ID:  240894.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2022-43903"
        }
    ],
    "metrics": [
        {
            "cvssV3_1": {
                "attackComplexity": "LOW",
                "attackVector": "NETWORK",
                "availabilityImpact": "LOW",
                "baseScore": 4.3,
                "baseSeverity": "MEDIUM",
                "confidentialityImpact": "NONE",
                "integrityImpact": "NONE",
                "privilegesRequired": "LOW",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
                "version": "3.1"
            },
            "format": "CVSS",
            "scenarios": [
                {
                    "lang": "en",
                    "value": "GENERAL"
                }
            ]
        }
    ],
    "references": [
        {
            "tags": [
                "vendor-advisory"
            ],
            "url": "https://www.ibm.com/support/pages/node/7030110"
        },
        {
            "tags": [
                "vdb-entry"
            ],
            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240894"
        }
    ]
}